telegram/loginurl.py

```diff
@@ -41,16 +41,24 @@ class LoginUrl(TelegramObject):
         url (:obj:`str`): An HTTP URL to be opened with user authorization data added to the query
             string when the button is pressed. If the user refuses to provide authorization data,
             the original URL without information about the user will be opened. The data added is
-            the same as described in Receiving authorization data.
-            NOTE: You must always check the hash of the received data to verify the authentication
-            and the integrity of the data as described in Checking authorization.
+            the same as described in
+            `Receiving authorization data <https://tinyurl.com/y3uqcgqv>`_
```

Comment:
please change both to use tha long url. I think it's better not to rely on third party services, where not strictly necessary …